R/gh-helpers.R

```diff
@@ -9,11 +9,11 @@ get_remote_url <- function(path, remote) {
   r <- git2r::repository(path, discover = TRUE)
   remote_names <- git2r::remotes(r)
   if (!length(remote_names)) {
-    stopc("Failed to lookup git remotes")
+    stop("Failed to lookup git remotes")
```

Comment:
`stopc()` was a helper function, gone now?

Reply:
It seems so, this is really just a quick fix for R CMD Check and imo should be added back in `f-27-automate` then?